src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs

```diff
@@ -175,11 +193,31 @@ public bool EditCallOffOrderingPartySectionDisplayed()
 			}
 		}
 
+		public bool CallOffOrderingPartyEnabled()
+		{
+			return Driver.FindElement(Pages.OrderForm.EditCallOffOrderingParty).TagName == "a";
+		}
+
+		public bool OrderDescriptionEnabled()
+		{
+			return Driver.FindElement(Pages.OrderForm.EditDescription).TagName == "a";
+		}
+
+		public bool SupplierInformationEnabled()
+		{
+			return Driver.FindElement(Pages.OrderForm.EditSupplier).TagName == "a";
```

Comment:
when would this fail, is the element expected to still be there but not a link? Or should this use findelements?

Reply:
it would return a false if not a link, so would fail the validation higher. it being a link signals that the section is enabled

Reply:
thats fine, as long as it doesn't fall over if the element itself can't be found. I'm expecting it to complete this method and return false